bin/ensemble.py

```diff
@@ -80,14 +46,14 @@ def load_data_files(args, params):
         testfile = h5py.File(os.path.join(params.dataset, args.testfile), 'r')
     else:
         raise ValueError('.npz or .h5 files are required; All sets must have the same format.')
-    
+
     files = [trainfile, validfile, testfile]
     return(files)
-    
-    
-    
+
```

Comment:
Oh no, my white spaces!! :( :(

Reply:
haha sorry about that. It was an accident.